docs/installation-and-operation/running-metabase-on-debian.md

```diff
@@ -1,144 +1,131 @@
+# MB_DB_TYPE=postgres
+# MB_DB_HOST=your_metabase_db_hostname
+# MB_DB_PORT=5432
+# MB_DB_DBNAME=your_metabase_db_name
+# MB_DB_USER=your_metabase_db_user
+# MB_DB_PASS=your_metabase_db_password
+EOF
 ```
+
+## Create a Metabase service
+
+Every service needs a configuration file that tells systemd how to manage it, and what capabilities it supports. System-wide services are typically registered at `/etc/systemd/system/<servicename>`. So, a Metabase service should live at `/etc/systemd/system/metabase.service`.
+
+### The Metabase service file
+
+The following command will create a file at `/etc/systemd/system/metabase.service` with a simple systemd service file to run Metabase:
+
+```sh
+sudo cat << EOF > /etc/systemd/system/metabase.service
 [Unit]
 Description=Metabase server
-After=syslog.target
 After=network.target
 
 [Service]
-WorkingDirectory=</your/path/to/metabase/directory/>
-ExecStart=/usr/bin/java --add-opens java.base/java.nio=ALL-UNNAMED -jar </your/path/to/metabase/directory/>metabase.jar
-EnvironmentFile=/etc/default/metabase
+WorkingDirectory=~
+ExecStart=/usr/bin/java --add-opens java.base/java.nio=ALL-UNNAMED -jar /home/metabase/metabase.jar
+EnvironmentFile=/home/metabase/.env
 User=metabase
 Type=simple
-StandardOutput=syslog
-StandardError=syslog
-SyslogIdentifier=metabase
 SuccessExitStatus=143
 TimeoutStopSec=120
 Restart=always
 
 [Install]
 WantedBy=multi-user.target
-```
-
-## Create syslog conf
-
-Next we need to create a syslog conf to make sure systemd can handle the logs properly.
-
-```
-sudo touch /etc/rsyslog.d/metabase.conf
-sudo <your-editor> /etc/rsyslog.d/metabase.conf
-
-if $programname == 'metabase' then /var/log/metabase.log
-& stop
-```
-
-Restart the syslog service to load the new config.
-
-```
-sudo systemctl restart rsyslog.service
-```
-
-## Environment variables for Metabase
-
-[Environment variables](../configuring-metabase/environment-variables.md) provide a good way to customize and configure your Metabase instance on your server. On Debian systems, services typically expect to have accompanying configs inside `etc/default/<service-name>`.
-
-### The Metabase config file
-
-Open your `/etc/default/metabase` environment config file in your editor:
-
-```
-sudo <your-editor> /etc/default/metabase
-```
-
-In `/etc/default/metabase`, replace configurable items (they look like `<some-var-name>`) with values sensible for your system. Some Metabase configs have available options, some of which are shown below, separated by `|` symbols:
-
-```
-MB_PASSWORD_COMPLEXITY=<weak|normal|strong>
-MB_PASSWORD_LENGTH=<10>
-MB_JETTY_HOST=<0.0.0.0>
-MB_JETTY_PORT=<12345>
-MB_DB_TYPE=<postgres|mysql|h2>
-MB_DB_DBNAME=<your_metabase_db_name>
-MB_DB_PORT=<5432>
-MB_DB_USER=<your_metabase_db_user>
-MB_DB_PASS=<ssshhhh>
-MB_DB_HOST=<localhost>
-MB_EMOJI_IN_LOGS=<true|false>
-# any other env vars you want available to Metabase
+EOF
 ```
 
 ## Final steps
```

Comment:
We should nix this final steps header, and promote the remaining headers to H2s (as "Final steps" doesn't add anything).